sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/search_space.py

```diff
@@ -14,7 +14,18 @@
 
 
 class SweepDistribution(ABC, RestTranslatableMixin):
-    def __init__(self, *, type: Optional[str] = None):  # pylint: disable=redefined-builtin
+    """Base class for sweep distribution configuration.
+
+    :param type: Type of distribution.
+    :type type: str
+    """
+
+    def __init__(self, *, type: Optional[str] = None) -> None:  # pylint: disable=redefined-builtin
+        """Sweep distribution configuration.
+
+        :param type: Type of distribution.
```

Comment:
Does this have a small list of acceptable values that's worth listing?

Reply:
Oh, I forgot about the uninstantiability... in that case it depends on how often you suspect that we'll need to be doing dev work using this class. If we're going to be call this often, then being able to just look at this to see the options all list in one place would be nice. But if this is likely to be relatively untouched code most of the time, then we can keep it concise.

For now I'll leave it as resolved. This isn't a choice that needs to be made now.